src/index.html

```diff
@@ -22,6 +22,401 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#index.html">
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="header__image"
+        />
+      </a>
+
+      <nav class="header__navigation">
+        <ul class="header__navigation-list">
+          <li class="header__navigation-item">
+            <a
+              href="#index.html"
+              class="header__navigation-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__navigation-item">
+            <a
+              href="#index.html"
+              class="header__navigation-link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__navigation-item">
+            <a
+              href="#index.html"
+              class="header__navigation-link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__navigation-item">
+            <a
+              href="#index.html"
+              class="header__navigation-link header__navigation-link--forth-one-blue"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__navigation-item">
+            <a
+              href="#index.html"
+              class="header__navigation-link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__navigation-item">
+            <a
+              href="#index.html"
+              class="header__navigation-link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__navigation-item">
+            <a
+              href="#index.html"
+              class="header__navigation-link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="header__navigation-item">
+            <a
+              href="#index.html"
+              class="header__navigation-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="cards-display">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="..\src\images\imac.jpeg"
```

Comment:
The image path uses backslashes (`\`). Consider using forward slashes (`/`) for web URLs to ensure compatibility across different environments.